javalanglib/src/main/scala/java/lang/Character.scala

```diff
@@ -519,6 +519,371 @@ object Character {
   @inline def compare(x: scala.Char, y: scala.Char): Int =
     x - y
 
+  // UnicodeBlock
+
+  class Subset protected (name: String) {
+    override def equals(that: Any): scala.Boolean = super.equals(that)
+    override def hashCode: scala.Int              = super.hashCode
+    override def toString(): String               = name
+  }
+
+  final class UnicodeBlock protected (name: String) extends Subset(name) {
+    private var start: Int = _
+    private var end: Int   = _
+    private def this(name: String, start: Int, end: Int) = {
+      this(name)
+      this.start = start
+      this.end = end
+    }
+  }
+
+  object UnicodeBlock {
+    private val allBlocks = new java.util.ArrayList[UnicodeBlock](293)
+    private val blocksByNormalizedName = new java.util.HashMap[String, UnicodeBlock]()
+
+    private def addUnicodeBlock(name: String, properName: String, start: Int, end: Int): UnicodeBlock =  {
+      val block = new UnicodeBlock(name, start, end)
+      allBlocks.add(block)
+
+      // Add normalized aliases
+      val lower = properName.toLowerCase
+      //   lowercase + spaces
+      blocksByNormalizedName.put(lower, block)
+      //   lowercase + no spaces
+      blocksByNormalizedName.put(lower.replaceAll(raw"\s", ""), block)
```

Comment:
The 2.11.x jobs on the CI are failing because 2.11 did not optimize `s"..."` and `raw"..."` interpolators at compile-time. That means that they use the Scala run-time library under the hood, which is not allowed in the javalanglib. We must therefore only use basic `"..."` string literals.